asset-transfer-private-data/chaincode-go/collections_config.json

```diff
@@ -0,0 +1,26 @@
+[
+ {
+   "name": "assetCollection",
+   "policy": "OR('Org1MSP.member', 'Org2MSP.member')",
+   "requiredPeerCount": 0,
+   "maxPeerCount": 3,
+   "blockToLive":1000000,
+   "memberOnlyRead": true
+},
+ {
+   "name": "Org1MSPDetailsCollection",
+   "policy": "OR('Org1MSP.member')",
+   "requiredPeerCount": 0,
+   "maxPeerCount": 3,
+   "blockToLive":3,
+   "memberOnlyRead": true
```

Comment:
In addition to the dissemination 'policy', we should also have a collection level 'endorsementPolicy'.
For an example see https://github.com/hyperledger/fabric/blob/master/integration/pvtdata/testdata/collection_configs/collections_config4.json.
Note, the endorsementPolicy is not needed on assetCollection, it can inherit the chaincode endorsement policy.
But of course Org2MSPDetailsCollection should have an endorsementPolicy as well.

Also, let's set memberOnlyWrite to false on the org specific collections, implying that one org client can write to the other orgs collection (in the case of a transfer).